### PR TITLE
Update for effect of limit on estimated row counts.

### PR DESCRIPTION
**Pull request by stbof**
DOC-1708

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm: 

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14600#-:-N7SvJiN7Y7M7uA3NOU7:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @stbof)

File: v22.1/explain.md [hunk 1]
```diff
@@ -136,6 +136,56 @@ The output also describes a set of properties, some global to the query, some sp
...
+~~~
+
+If you then limit the number of returned rows:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+EXPLAIN SELECT * FROM rides WHERE revenue > 90 ORDER BY revenue ASC limit 10;
+~~~
+
+The limit is reflected both in the estimated row count and a `limit:` property:
```
Comment (taroface) [inline v22.1/explain.md:173]: Does `limit:` need the colon?

**Review by taroface — APPROVED** [review]
LGTM, 1 question

File: v22.1/explain.md (hunk 1 above)
Comment (stbof) [reply v22.1/explain.md:173]: Good point. No. Will remove it.

Comment (stbof) [inline v22.1/explain.md:173]:
```suggestion
The limit is reflected both in the estimated row count and a `limit` property:
```

**Pull request merged**